src/verify/hooks.ts

```diff
@@ -107,6 +111,8 @@ export function useVerifyPhoneNumber(phoneNumber: string, countryCallingCode: st
           phoneNumber,
           clientPlatform: Platform.OS,
           clientVersion: DeviceInfo.getVersion(),
+          clientBundleId: DeviceInfo.getBundleId(),
+          publicDataKey,
```

Comment:
nit: I'm using this name on the backend:
```suggestion
          publicDataEncryptionKey,
```